modules/rosa-sts-required-roles-and-policies.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// rosa_architecture/rosa-sts-about-iam-resources.adoc
+//
+:_content-type: CONCEPT
+[id="rosa-sts-required-roles-and-policies_{context}"]
+= Configuring ROSA cluster roles and policies
```

Comment:
I think this content needs to be worked into the existing separate concept sections about the account-wide roles and policies, the cluster-specific operator IAM roles and the OIDC provider. The headings of those existing sections might need changing, but the different resources I think should remain as separate sections so that the topics can be addressed separately.

We can work the fact that the OCM and user roles enable OCM to create the resources for you, but that really needs to be positioned as OCM and UI options in the same sections.